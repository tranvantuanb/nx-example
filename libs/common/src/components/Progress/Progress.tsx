import React from 'react';
import {
  Progress as AntdProgress,
  ProgressProps as AntdProgressProps,
} from 'antd';

export type ProgressProps = AntdProgressProps;

export const Progress: React.FC<ProgressProps> = React.memo((props) => {
  return <AntdProgress {...props} />;
});
