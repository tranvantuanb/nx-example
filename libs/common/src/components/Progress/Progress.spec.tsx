import { render } from '../../test';

import { Progress, ProgressProps } from './Progress';

describe('Progress', () => {
  it('should render successfully', () => {
    const defaultProps: ProgressProps = {};
    const { container } = render(<Progress {...defaultProps} />);
    expect(container).toBeTruthy();
    expect(container).toMatchSnapshot();
  });
});
