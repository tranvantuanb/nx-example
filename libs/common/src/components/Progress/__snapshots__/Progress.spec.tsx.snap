// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Progress should render successfully 1`] = `
<div>
  <div
    class="ant-progress ant-progress-line ant-progress-status-normal ant-progress-show-info ant-progress-default"
  >
    <div
      class="ant-progress-outer"
    >
      <div
        class="ant-progress-inner"
      >
        <div
          class="ant-progress-bg"
          style="width: 0%; height: 8px;"
        />
      </div>
    </div>
    <span
      class="ant-progress-text"
      title="0%"
    >
      0%
    </span>
  </div>
</div>
`;
